src/configure/configure.ts

```diff
@@ -173,7 +173,11 @@ class Orchestrator {
                 this.inputs.azureSession = getAzureSession();
             }
             const repoAnalysisResult = await this.getRepositoryAnalysis();
-            await this.getSelectedPipeline(repoAnalysisResult);
+            if (this.context['isResourceAlreadySelected'] && resourceNode.type === TargetResourceType.AKS) {
+                await this.getFilteredTemplates("docker", "Azure:AKS");
```

Comment:
This will break the azure pipelines flow. `getSelectedPipeline` handles both local templates (which I thin is used for azure pipelines case) and remote templates (which is used for gh workflow case). It decides the kind of template to use based on the repo provider and some extension settings.
Hence it would not be wise to bifurcate here. We need to go deeper inside getSelectedPipeline and then only call getFilteredTemplates in case of remote templates.

Reply:
ohh! yes! 
will make the required changes.